Makefile.libcompat

```diff
@@ -92,10 +92,13 @@ build${libcompat}: .PHONY
 	    -p ${WORLDTMP}/usr/lib/debug/usr >/dev/null
 .endif
 # XXX: Remove this hack once morello-llvm supports lib64c
+# NB: bsd.compat.mk is a no-op for cleandir, so handle here
```

Comment:
Alternatively, I could pull out the one commit from my varargs branch that removes the libcheri fallback since morello-llvm has had the lib64c support since about December or so.  If we merge that one part we won't need this change?  I'm fine with doing it in either order though, but if I merge the libcheri thing now we won't need to add a revert of this to that commit.

Reply:
We could; I've just had this sitting around for several months now in my local benchmarking branch, ran into it very early on and forgot about it. I don't particularly mind, other than that this branch exists and is ready to merge now rather than picking that out (and either adding libcheri to obsolete files as commented there or just extracting the worldtmp bit, which seems a little odd).

Reply:
Having said that, it might be nice to get rid of libcheri before the release and avoid confusion anyway